aea/test_tools/test_contract.py

```diff
@@ -66,19 +107,15 @@ def contract(self) -> Contract:
         return value
 
     @classmethod
-    def setup(cls, **kwargs: Any) -> None:
```

Comment:
Setup gets called before each test method call. Defining it as a class method is confusing and leads to misuse.